app/models/problem_in_quiz.py

```
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, ForeignKey, TIMESTAMP
```

Ruff (0.8.2)

`sqlalchemy.TIMESTAMP` imported but unused

Remove unused import: `sqlalchemy.TIMESTAMP`

(F401)